server/db/db.js

```diff
@@ -1,10 +1,11 @@
 const Sequelize = require('sequelize')
-const pkg = require('../../package.json')
+// const pkg = require('../../package.json')
 
 // const databaseName = pkg.name + (process.env.NODE_ENV === 'test' ? '-test' : '')
 
 const db = new Sequelize(
-  process.env.DATABASE_URL || `postgres://localhost:5432/tyckets`,
+  process.env.DATABASE_URL ||
+    `postgres://kevinminutti:minutti64@localhost:5432/tyckets`,
```

Comment:
this might cause the rest of us problems maybe? since we dont have your user kevinminutti

Reply:
Yes can please remove the difference